### ISSUE TITLE
Archive Repo

### ISSUE THREAD

**Issue by Windyo** [jamalneufeld/ghostHunter#112]
@jamwise new official search feature means we should archive this i think.

**Comment by hsarji** [comment]
I don't think that is a good idea since native search [has limits](https://github.com/jamwise/ghostHunter/pull/111#issuecomment-1184608684).

**Comment by Windyo** [comment]
I mean the GhostHunter search has limits as well, including but not limited to the fact that its main maintainer for the past four years (me) doesn't actually know Javascript and doesn't even use Ghost anymore...

**Comment by hsarji** [comment]
@Windyo Thank you for your honest reply! If you're the main maintainer for the past four years and you don't actually know JavaScript and don't even use Ghost anymore, then I withdraw my objection.

**Comment by Windyo** [comment]
Yeah, in short I just stumbled upon GhostHunter when Ghost published its internal API at the time.

The official API versio was v0.1, there wasn't content api keys or anything. GH used RSS at that point to fetch the posts and that started to slow down starting at 100 posts, and got worse progressively as to be nigh unusable after something like 200.

My main contribution was learning just enough javascript based on my BASH skills to see how to call the API from GH and shove that into the lunr array in my own fork. I opened a PR thereafter.
@jamwise didn't support the extension anymore and added me as a contributor.

Over the years I did a bit of maintenance here and there, added functions as I needed them (description parsing etc), followed Ghost versions, added merges from people that followed code styles, but that's pretty much it. If you check forks, a few people have done refactoring which was sorely needed.

I stopped using Ghost two years ago when I completely stopped blogging, and my career path never needed me to really learn Javascript, so in effect yes - GhostHunter has been pretty much fully maintained by a person that doesn't know JS for the past 4 years. :)

Yes, I know no one cares, but it's not a story that I can break out at parties :D
